src/NuGet.Services.V3Repository/V3PathCalculator.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.V3Repository
+{
+    public class V3PathCalculator

```

Comment:
a single place that knows how to format registration/flat container paths. Used to generate Index and query results.